exercise/sitemaps.py

```diff
@@ -1,14 +1,14 @@
-from django.contrib import sitemaps
 from django.urls.base import reverse
 from django.utils import timezone
 
 from course.models import CourseInstance
+from lib.sitemaps import AplusSitemap
 from .models import BaseExercise, CourseChapter, LearningObject
 
 
-class BaseExerciseSitemap(sitemaps.Sitemap):
-    priority = 0.2
+class BaseExerciseSitemap(AplusSitemap):
     changefreq = 'daily'
+    base_priority = 0.1
```

Comment:
A+ supports "independent" exercises that are not embedded in chapters. They should have the same maximum priority as chapter pages. If the `items()` method only returned BaseExercises with `status = READY`, then this class would focus on "independent" exercises.

Reply:
See the above comment. ``BaseExerciseSitemap`` is for "independent" exercises that are not embedded in chapters and also for exercises that are embedded in chapters. ``HiddenBaseExerciseSitemap`` is an attempt at hiding the standalone exercise pages of exercises that are embedded in chapters from Google search, since they could previously appear in Google, even though they were not part of the sitemap.